File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinShieldconexMgmt SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinShieldconexMgmtMakeContext
{
    public static function call(array $ctxmap, ?BluefinShieldconexMgmtContext $basectx): BluefinShieldconexMgmtContext
    {
        return new BluefinShieldconexMgmtContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# BluefinShieldconexMgmt SDK utility: make_context
require_relative '../core/context'
module BluefinShieldconexMgmtUtilities
  MakeContext = ->(ctxmap, basectx) {
    BluefinShieldconexMgmtContext.new(ctxmap, basectx)
  }
end
